doc/source/how-to/setting-up.rst

```diff
@@ -467,6 +467,41 @@ For information on setting up ``SSH`` with ``GitHub``, in the ``GitHub`` documen
 see `Connecting to GitHub with SSH
 <https://docs.github.com/en/authentication/connecting-to-github-with-ssh>`_.
 
+Handling line endings
+~~~~~~~~~~~~~~~~~~~~~
+Every time you introduce a new line by pressing the `return` key, an invisible
```

Comment:
Great sentence. My only question is the key name and formatting of it. My English keyboard says "enter" so traditional Ansys doc would say ... by pressing the **Enter** key. I'm not sure if **Enter** is true for keyboards for other languages though! In PyAnsys doc, we are using bold for any UI elements and key names.

Reply:
Then, let's use the "Enter" name for this.

Reply:
+1 for "Enter".